pkg/registry/registrytest/etcd.go

```diff
@@ -222,6 +222,9 @@ func (t *Tester) emitObject(obj runtime.Object, action string) error {
 			return err
 		}
 		_, _, err = t.storage.Delete(ctx, accessor.GetName(), nil)
+		if err != nil {
```

Comment:
This `err` will be caught by `return err` in #L232, so seems like it's not necessary to check it here. WDYT?